small/obuf.c

```diff
@@ -98,6 +98,7 @@ obuf_destroy(struct obuf *buf)
 #ifndef NDEBUG
 	obuf_create(buf, buf->slabc, buf->start_capacity);
 #endif
+	buf->slabc = NULL;
```

Comment:
Please drop `obuf_create` and simply use `obuf_clear` (which would zero out the struct). The old code should work fine this way.

Reply:
Done. See the previous comment for details.